SkyEditor.RomEditor.Rtdx.ConsoleApp/Scripts/Utilities/Rtdx/GenerateTranscript.csx

```diff
@@ -0,0 +1,67 @@
+// This script generates a text file containing all dialogues.
+// The resulting transcript.txt file will be saved in the current working directory.
+
+#load "../../../Stubs/RTDX.csx"
+
+using System;
+using System.IO;
+using SkyEditor.RomEditor.Domain.Rtdx.Constants;
+using SkyEditor.RomEditor.Domain.Rtdx;
+using SkyEditor.RomEditor.Domain.Rtdx.Structures;
+using System.Text.RegularExpressions;
+using System.Text;
+
+var rtdxRom = (RtdxRom) Rom;
```

Comment:
`Rom` is an IRtdxRom which should have everything you need. It'd be better to add anything missing to the interface than to cast to this instance.